phobos/scripts/smurfs_in_pybullet.py

```diff
@@ -161,17 +165,21 @@ def parse_smurfs(client_id, path, basedir=os.getcwd(), loaded=None):
                 if entity["anchor"] == "world" or entity["anchor"] == "parent" and entity["parent"] == "world":
                     load_dict["useFixedBase"] = 1
                 elif entity["anchor"] == "parent" and entity["parent"] != "world":
-                    print("WARNING: PyBullet can't create a joint for the parent anchoring in entity", entity["file"])
+                    entity_file = entity["file"]
+                    log.warning(f"WARNING: PyBullet can't create a joint for the parent anchoring in entity {entity_file}")
             load_dict["id"] = pb.loadURDF(**load_dict)
             if info is not None:
                 load_dict["info"] = info
             if "name" in entity.keys():
                 if entity["name"] not in loaded.keys():
                     loaded[entity["name"]] = deepcopy(load_dict)
                 else:
-                    print("WARNING: The entity name", entity["name"], "of", entity["file"], "is a duplicate!")
+                    entity_name = entity["name"]
+                    entity_file = entity["file"]
```

Comment:
Why don't you take those directly?

Reply:
Pycharm was marking it because it didn't like the  " in the brackets. Fixing it